graphql/resolvers.go

```diff
@@ -2412,15 +2412,16 @@ func (r *queryResolver) BuildBaron(ctx context.Context, taskID string, exec int)
 	execString := strconv.Itoa(exec)
 
 	searchReturnInfo, bbConfig, err := GetSearchReturnInfo(taskID, execString)
+	if err != nil {
+		return nil, InternalServerError.Send(ctx, err.Error())
+	}
 	if !bbConfig.ProjectFound || !bbConfig.SearchConfigured {
```

Comment:
You mentioned that for projects that don't use build baron, we don't want to send them errors. Is this going to happen now that we actually return errors? Should we case on "Build baron project not found", or not return an error from GetSearchReturnInfo in that case?

Reply:
I reconfigured a bit, I think this is no longer a concern.

Reply:
This looks good! now that we no longer error check out of order, can we just set `BuildBaronConfigured = bbConfig.ProjectFound && bbConfig.SearchConfigured` rather than bother with an if statement? (Since we return BuildBaron with the SearchReturnInfo either way)

Reply:
good point, fixed